index.html

```diff
@@ -6,15 +6,164 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <!-- Here we import the Boostrap CSS stylesheet -->
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous">
-    <!-- Add a link to your custom CSS here -->
+    <link href="css/main.css" rel="stylesheet" type="text/css">
 
     <title>Responsive grid project</title>
 </head>
 
+
 <body>
+<div id="page">
+  
+    <header class="container">
+        <div class="row">
+            <nav class="navbar navbar-expand-sm fixed-top">
+
+                <div class="col-12 col-sm-4">
```

Comment:
To follow correct use of Bootstrap, the `col-...` elements need to be direct children of `row`. Here, the `nav` is in between which is breaking that direct relationship. 
The reason for this is that `row` uses `display: flex`, which only works on its direct (i.e. immediate) children.